src/game/engine/tab.h

```diff
@@ -79,7 +79,7 @@ class TabClass : public SidebarClass
 
 public:
     TabClass();
-    TabClass(const NoInitClass &noinit) : SidebarClass(noinit), CreditDisplay(noinit), TimerFlashTimer(noinit), CreditsFlashTimer(noinit) {}
```

Comment:
Again, this needs to call noinit ctors or the default ones get called which we do not want.